components/BCRegistriesAccount.vue

```diff
@@ -201,25 +201,15 @@
                 are currently available to search. Other business types will be added in the future.
```

Comment:
Not sure if this was in your ticket but this tooltip text is wrong I think? People can search for all types of businesses

Reply:
It is not mentioned in the ticket. I will ask Janis